aptos-move/framework/src/extended_checks.rs

```diff
@@ -148,7 +148,7 @@ impl<'a> ExtendedChecker<'a> {
                 if !ok {
                     self.env.error(
                         &fun.get_id_loc(),
-                        "`init_module` function can only take signers as parameters",
+                        "`init_module` function can only take `signer` values and immutable references as parameters",
```

Comment:
should be `signer` or the reference to a `signer` according to the check above?

Reply:
must be immutable reference, I figured out how to word it better.